x-pack/test/functional_with_es_ssl/apps/cases/upgrade.ts

```diff
@@ -0,0 +1,288 @@
+  getExternalServiceSimulatorPath,
+  ExternalServiceSimulator,
+} from '../../../alerting_api_integration/common/fixtures/plugins/actions_simulators/server/plugin';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const cases = getService('cases');
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+
+  const updateConnector = async (id: string, req: Record<string, unknown>) => {
+    const { body: connector } = await supertest
+      .put(`/api/actions/connector/${id}`)
+      .set('kbn-xsrf', 'true')
+      .send(req)
+      .expect(200);
+
+    return connector;
+  };
+
+  const replaceNewLinesWithSpace = (str: string) => str.replace(/\n/g, ' ');
+
+  /**
+   * This test test an upgrade from 7.15 to the latest Kibana version.
+   * The x-pack/test/functional/fixtures/kbn_archiver/cases/7.17.5/case.json contains a case with
+   * all available user actions except user actions related to alerts. By importing the case,
+   * all migrations run. We ensure that the case shows all data after migrations.
+   */
+
+  describe('Upgrade', function () {
+    const CONNECTOR_ID = '92b970e0-09c4-11ed-9088-a95ba44d7455';
+    const CASE_ID = 'e26361b0-09c3-11ed-9088-a95ba44d7455';
+
+    before(async () => {
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/cases/7.17.5/case.json'
+      );
+
+      await cases.testResources.installKibanaSampleData('logs');
+
+      const jiraSimulatorURL = kibanaServer.resolveUrl(
+        getExternalServiceSimulatorPath(ExternalServiceSimulator.JIRA)
+      );
+
+      await updateConnector(CONNECTOR_ID, {
+        name: 'Jira',
+        config: {
+          apiUrl: jiraSimulatorURL,
+          projectKey: 'ROC',
+        },
+        secrets: { email: 'elastic@elastic.co', apiToken: '123' },
+      });
+
+      await cases.navigation.navigateToSingleCase('cases', CASE_ID);
+    });
+
+    after(async () => {
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/fixtures/kbn_archiver/cases/7.17.5/case.json'
+      );
+
+      await cases.testResources.removeKibanaSampleData('logs');
+    });
+
+    describe('Case view page', function () {
+      it('shows the title correctly', async () => {
+        const title = await testSubjects.find('header-page-title');
+        expect(await title.getVisibleText()).equal('Upgrade test in Kibana');
+      });
+
+      it('shows the description correctly', async () => {
+        const desc = await find.byCssSelector(
+          '[data-test-subj="description-action"] [data-test-subj="user-action-markdown"]'
+        );
+
+        expect(await desc.getVisibleText()).equal(`Testing upgrade! Let's see how it goes.`);
+      });
+
+      it('shows the description correctly', async () => {
```

Comment:
I think this one is a duplicate of the test above right?

Reply:
Yeap 🙂. I will remove it.